plugins/ocp/ocp-telemetry-decode.c

```diff
@@ -126,6 +132,19 @@ void print_telemetry_fifo_event(__u8 class_type,
 
 			printf("  CSTS Reg Data : 0x%08x\n",
 					le32_to_cpu(csts_reg_data));
+		} else if (id == OOB_COMMAND) {
+			printf("  Cmd Op Code   : 0x%02x\n", data[0]);
+			__u16 status = *(__u16 *)(data + 1);
```

Comment:
You need to use the same memcpy trick as above, this likely to work on platforms which enforce data alignment.

The rest looks good.